CefSharp.BrowserSubprocess.Core/ConcurrentObjectRegistry.h

```diff
@@ -0,0 +1,53 @@
+﻿// Copyright © 2010-2015 The CefSharp Project. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style license that can be found in the LICENSE file.
+
+#pragma once
+
+#include "JavascriptCallbackWrapper.h"
+
+using namespace System::Collections::Concurrent;
+
+namespace CefSharp
+{
+    namespace Internals
+    {
+        generic <typename T>
+        public ref class ConcurrentObjectRegistry
+        {
+        private:
+            bool _deleteEntriesOnDestroy;
+            Int64 _lastId;
+            ConcurrentDictionary<Int64, T>^ _entries;
+
+        public:
+            ConcurrentObjectRegistry(bool deleteEntriesOnDestroy) :
```

Comment:
> and if the developer wants to automatically delete all contained objects he can set this to true.

Which developer exactly? This code isn't exposed to the public.

> As it is currently only used with true it can also be removed but I thought of this class as a general data structure.

I'd rather not have code that's unused just lying around as a **maybe**. Please remove.